core/client/fs/src/main/java/alluxio/client/block/stream/LocalFilePacketReader.java

```diff
@@ -155,6 +156,9 @@ public void close() throws IOException {
       try {
         NettyRPC.call(NettyRPCContext.defaults().setChannel(mChannel).setTimeout(READ_TIMEOUT_MS),
             new ProtoMessage(request));
+      } catch (Exception e) {
+        mChannel.close();
```

Comment:
Since this is async, is there a possibility that the close does not happen until it is require again as a healthy resource?